File: TaskMasterAPI/src/main/java/com/devlab/taskmaster/dto/task/TaskDTO.java
package com.devlab.taskmaster.dto.task;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class TaskDTO {

    private Integer id;

    private String name;

    private String status;

    private String priority;

    private TaskProjectDTO project;

    private TaskUserDTO assignee;
}

File: TaskMasterWeb/TaskMasterWeb.Client/Dtos/Task/TaskDto.cs
namespace TaskMasterWeb.Client.Dtos.Task;

public record TaskDto
{
    public int Id { get; set; }

    public string Name { get; set; } = default!;

    public string Status { get; set; } = default!;

    public string Priority { get; set; } = default!;

    public TaskProjectDto Project { get; set; } = default!;

    public TaskUserDto Assignee { get; set; } = default!;
}